components/organisms/RepositoriesCart/repositories-cart.tsx

```diff
@@ -54,64 +55,69 @@ const RepositoriesCart = ({
+       Repositories Added
+      </Title>
+      <div className="w-full lg:w-[364px] border flex flex-col gap-2 rounded-lg p-6">
+        {/* Empty state of Cart */}
+        {cartItems.length > 0 && hasItems ? (
+          cartItems.map((item, index) => (
+            <div className="flex flex-col w-full" key={`${index}/${Math.random()}`}>
+              {item}
+            </div>
+          ))
+        ) : (
+          <div className="flex flex-col items-center w-full gap-2 py-4">
+            <CartIllustration classNames="-translate-x-2" />
+            <CartIllustration classNames="-translate-x-10" />
+            <CartIllustration classNames="translate-x-4" />
+            <Text className="mt-3 !text-base">You haven’t added repositories yet.</Text>
           </div>
-        ))
-      ) : (
-        <div className="flex flex-col items-center w-full gap-2 py-4">
-          <CartIllustration classNames="-translate-x-2" />
-          <CartIllustration classNames="-translate-x-10" />
-          <CartIllustration classNames="translate-x-4" />
-          <Text className="mt-3 !text-base">You haven’t added repositories yet.</Text>
-        </div>
-      )}
-      {!!history && (
-        <div className={`${history.length > 0 && "border-1"} mt-2 py-1`}>
-          {history.length > 0 ? <Text>Add again:</Text> : ""}
-          {history.length > 0 &&
-            history.map(({ orgName, repoName, totalPrs, avatar }, index) => (
-              <div key={`${index}/${orgName}/${repoName}`} className="flex items-center justify-between mt-2">
-                <div className="flex items-center gap-3 ">
-                  <Avatar avatarURL={avatar} initials="" size="sm" className="" />
-                  <Text className="!text-sm !text-light-slate-11">
-                    {orgName} / <span className="text-light-slate-12">{repoName}</span>
-                  </Text>
+        )}
+        {!!history && (
+          <div className={`${history.length > 0 && "border-1"} mt-2 py-1`}>
+            {history.length > 0 ? <Text>Add again:</Text> : ""}
+            {history.length > 0 &&
+              history.map(({ orgName, repoName, totalPrs, avatar }, index) => (
+                <div key={`${index}/${orgName}/${repoName}`} className="flex items-center justify-between mt-2">
+                  <div className="flex items-center gap-3 ">
+                    <Avatar avatarURL={avatar} initials="" size="sm" className="" />
+                    <Text className="!text-sm !text-light-slate-11">
+                      {orgName} / <span className="text-light-slate-12">{repoName}</span>
+                    </Text>
+                  </div>
+                  <div className="flex items-center gap-2 text-sm text-light-slate-10">
+                    <button
+                      onClick={() => onAddToCart(`${orgName}/${repoName}`)}
+                      className="border text-xs flex items-center px-[6px] p-[3px] rounded-md"
```

Comment:
```suggestion
                      className="border text-xs flex items-center px-1.5 p-0.5 rounded-md"
```

`p-0.5` -> 2px  || p-1 -> 4px ??

Reply:
@NsdHSO brings up some a good point. Should we lean into the design system here?

Reply:
Thanks for that, I think going with design system is much better, but I did not like to touch this parts since it would be out of scope of the main issue.